eth2near/eth2near-block-relay/index.js

```diff
@@ -403,43 +170,45 @@ class Eth2NearRelay {
   }
 }
 
-async function runEth2NearRelay () {
-  const masterAccount = RainbowConfig.getParam('near-master-account')
-  const masterSk = RainbowConfig.getParam('near-master-sk')
-  const keyStore = new nearAPI.keyStores.InMemoryKeyStore()
-  await keyStore.setKey(
-    RainbowConfig.getParam('near-network-id'),
-    masterAccount,
-    nearAPI.KeyPair.fromString(masterSk)
-  )
-  const near = await nearAPI.connect({
-    nodeUrl: RainbowConfig.getParam('near-node-url'),
-    networkId: RainbowConfig.getParam('near-network-id'),
-    masterAccount: masterAccount,
-    deps: {
-      keyStore: keyStore
-    }
-  })
-
-  const relay = new Eth2NearRelay()
-  const clientContract = new EthOnNearClientContract(
-    new nearAPI.Account(near.connection, masterAccount),
-    RainbowConfig.getParam('near-client-account')
-  )
-  await clientContract.accessKeyInit()
-  console.log('Initializing eth2near-relay...')
-  relay.initialize(clientContract, RainbowConfig.getParam('eth-node-url'))
-  console.log('Starting eth2near-relay...')
-  relay.run()
-}
+// TODO: remove it once confurmed it is not used
+//
+// async function runEth2NearRelay () {
+//   const masterAccount = RainbowConfig.getParam('near-master-account')
+//   const masterSk = RainbowConfig.getParam('near-master-sk')
+//   const keyStore = new nearAPI.keyStores.InMemoryKeyStore()
+//   await keyStore.setKey(
+//     RainbowConfig.getParam('near-network-id'),
+//     masterAccount,
+//     nearAPI.KeyPair.fromString(masterSk)
+//   )
+//   const near = await nearAPI.connect({
+//     nodeUrl: RainbowConfig.getParam('near-node-url'),
+//     networkId: RainbowConfig.getParam('near-network-id'),
+//     masterAccount: masterAccount,
+//     deps: {
+//       keyStore: keyStore
+//     }
+//   })
+
+//   const relay = new Eth2NearRelay()
+//   const clientContract = new EthOnNearClientContract(
+//     new nearAPI.Account(near.connection, masterAccount),
+//     RainbowConfig.getParam('near-client-account')
+//   )
+//   await clientContract.accessKeyInit()
+//   console.log('Initializing eth2near-relay...')
+//   relay.initialize(clientContract, RainbowConfig.getParam('eth-node-url'))
+//   console.log('Starting eth2near-relay...')
+//   relay.run()
+// }
```

Comment:
I use it here: https://github.com/near/nearcore/pull/3535/files#diff-775a9f1990e68594074cb25791d9d5cc0c08ac783767390127fe66e6e01c17a7R114

Reply:
@Kouprin Do you plan to pass any arguments to it? Can we avoid using RainbowConfig there and expect all the arguments passed explicitly?

Reply:
@frol it couldn't avoid config usage, otherwise it needs to implement its own config. :) However, this code may be removed in case of having the way of starting eth2near-relay with config and no pm2 coverage.

BTW, using "make-runnable" seems very convenient way for starting services. Basically, I'd suggest to use cli "relay start" as wrapper of this code.

Reply:
@Kouprin make-runnable automatically parses all the CLI arguments: https://github.com/super-cache-money/make-runnable. I think you can just:

```
self.pid.value = subprocess.Popen(['node', eth2near_block_relay_path, 'runEth2NearRelay', '--ethNodeUrl', ethNodeUrl, ...], stdout=self.stdout, stderr=self.stderr).pid
```

That is, explicitly pass the config values as CLI args.

Reply:
@frol not sure what are you talking about. How to extract necessary config args to pass them explicitly in python code?

Reply:
@Kouprin the new design is that it does not automagically save a hidden config/state. As a specific example, `StartEth2NearRelayCommand.execute` expects all the parameters get via input arguments. Only `cli` package is aware of how to read and save the RainbowConfig file, all the rest codebase is using the method arguments and return values as its API, which `cli` is complying with.

Reply:
Can you just use `cli` (`start eth2near-relay`) instead of making a separate CLI out of the package?

Reply:
Automagically :)
> Can you just use cli (start eth2near-relay) instead of making a separate CLI out of the package?
Yes. However, it's shouldn't be forcibly covered with pm2. Not sure is it possible to run without pm2 right now.

Reply:
@Kouprin yes, it is possible, just pass `--daemon false`. Please, give it a try, and if that plays well for you, I will remove make-runnable setup.

Reply:
Could we please invert daemon usage to start with `--daemon false` by default? I expect we should never use daemon and get rid of it in future completely.

Reply:
Let's do it in a separate PR. It will require to change the README and tests. I don't want to squash more changes than necessary into a single PR